http-server/src/main/java/io/airlift/http/server/HttpServer.java

```diff
@@ -291,6 +291,9 @@ public HttpServer(HttpServerInfo httpServerInfo,
                 }
                 sslContextFactory.setSecureRandomAlgorithm(config.getSecureRandomAlgorithm());
                 sslContextFactory.setWantClientAuth(true);
+                // Since Jetty 9.4.12 the list of excluded cipher suites includes "^TLS_RSA_.*$" by default.
+                // We reset that list here to enable use of those cipher suites.
+                sslContextFactory.setExcludeCipherSuites();
```

Comment:
This updates the `sslContextFactory` for the `adminConnector`. The `sslContextFactory` for the `httpsConnector` is already updated from the config on L238, so we don't need to change that.

Reply:
I think it would make more sense to add the exclude/include cipher calls L238 here.  My guess is we just missed the admin connector when adding that code.